src/components/pages/MatchUp/RenderMatchUp.js

```diff
@@ -11,139 +9,125 @@ import { modalInstructions } from '../../../utils/helpers';
 
 const RenderMatchUp = props => {
   const { push } = useHistory();
-  const [faceoffs, setFaceoffs] = useState([]);
-  const [modalVisible, setModalVisible] = useState(true);
-
-  const goToPointShare = () => {
-    push('/child/point-share');
-  };
 
-  useEffect(() => {
-    if (props.child.VotesRemaining <= 9) {
-      setModalVisible(false);
-    }
-    setFaceoffs(props.faceoffs);
-  }, [props]);
-  const handleVote = e => {
+  const goToPointShare = e => {
     e.preventDefault();
-    push('/child/match-up/squad-vote');
-  };
-  const back2Dash = e => {
-    e.preventDefault();
-    push('/child/dashboard');
```

Comment:
nice change on making sure the function names are correct to the function being done